.github/workflows/build.yaml

```diff
@@ -46,10 +61,45 @@ jobs:
         PYTHON: python
     - name: Install dependencies
       run: make setup
+    - name: Create changelog and bump
+      if: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
This ensures that the step runs only when the target is the `main` branch?

Should we maybe also consider checking [`github.event.type`](https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types#event-object-common-properties)?

Reply:
Hmm, so if somebody uses this Action on a repo where the default branch has a different name (e.g. `master`) they’d have to make changes. Hmm.

Reply:
As per conversation, let’s leave a note in the README that this template repository assumes `main` to be the default branch.